README.md

```diff
@@ -216,6 +216,28 @@ void Example()
 ## Discovery V2
 Discovery v2 is only available on Cloud Pak for Data.
 
+## Transaction IDs
+Every call from the SDK will receive a response which will contain a transaction ID, accessible via the x-global-transaction-id header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
```

Comment:
The wording at the start of this is a bit confusing to me. I feel like this might be more clear:

```
Every SDK call will return a response...
```

Reply:
I think this is fine language, too. I just grabbed this from the [cloud sdk templates repo](https://github.ibm.com/CloudEngineering/java-sdk-template#transaction-ids). Do you think its worth getting it changed throughout? 

Regarding doc page - @SirSpidey doesn't know of anything. I'll see if I can find something about getting a transactionId for support tickets.

Reply:
well, that is a doozy of a sentence and it breaks a number of IBM style guidelines. So my answer is yes, get it changed.